components/library/glib/patches/04-xopen-version.patch

```diff
@@ -1,6 +1,11 @@
---- glib-2.62.3/meson.build.1	2019-12-18 21:58:02.109823287 +0000
-+++ glib-2.62.3/meson.build	2019-12-18 21:59:14.389494095 +0000
-@@ -2011,8 +2011,6 @@
+Remove Unix standard settings for Solaris 11.4, so we don't exclude all
```

Comment:
Solaris 11.4?  Really? :-)

Reply:
The patch is borrowed from solaris-userland, so I will keep the comment.